tests/kuttl/tests/deployments/01-deploy-openstack.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      oc apply -n openstack -f ../../../../config/samples/core_v1beta1_openstackcontrolplane.yaml
```

Comment:
As @abays pointed out, we should replace `-n openstack` with `-n $NAMESPACE`, so we can run the tests in a different namespace